paperboy-cli/src/cli.rs

```diff
@@ -20,8 +20,10 @@ pub enum Commands {
         email: String,
         /// Subscription file
         subscription_file: String,
-        /// Email template (Using handlebars)
-        template: String,
+        /// Email template as HTML (Using handlebars)
+        template_html: String,
+        /// Email template as text (Using handlebars)
+        template_text: Option<String>,
```

Comment:
Instead of adding an extra option to the CLI, could we just assume that the text file will be named `$template_html + _text.hbs`?  + some short explanation in the readme file.

Reply:
Yes, but I find this a bit sloppy
And will lead to more complications than leaving the user choose ?

Do you really want this change ?

Reply:
yeah, I thought about it today. 
I think you are right. Let's keep the option.

Reply:
Awesome!